packages/lesswrong/components/sunshineDashboard/SunshineCuratedSuggestionsList.tsx

```diff
@@ -94,13 +108,15 @@ const SunshineCuratedSuggestionsList = ({ terms, belowFold, classes, setCuration
     }
   }
 
+  const needsDraftsText = !timeForCuration && !hasDrafts ? " (No drafts!)" : "";
+
   const { SunshineListTitle, SunshineCuratedSuggestionsItem, MetaInfo, FormatDate,
     LoadMore, LWTooltip, ForumIcon } = Components
 
   return (
     <div className={classNames(classes.root, statusClass)}>
       <SunshineListTitle>
-        <Link to={`/admin/curation`}>Suggestions for Curated</Link>
+        <Link to={`/admin/curation`}>Suggestions for Curated{needsDraftsText}</Link>
```

Comment:
🤔 I'm a bit surprised that this kind of inlining works... I'd have expected you to need to wrap all of the inner text in curly braces and then backticks, or something.

Reply:
There's an example of doing this on the Quick Start page of the React docs
<img width="1017" alt="image" src="https://github.com/user-attachments/assets/a1ac28cd-c252-433a-8471-dd47c6bd643a" />